psi4/src/read_options.cc

```diff
@@ -999,6 +999,15 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_double("SAPT_DFT_GRAC_SHIFT_B", 0.0);
         /*- Compute the Delta-HF correction? -*/
         options.add_bool("SAPT_DFT_DO_DHF", true);
+        /*- Enables the hybrid xc kernel in dispersion? !expert -*/
+        options.add_bool("SAPT_DFT_DO_HYBRID", true);
+        /*- Scheme of exchange-dispersion? !expert -*/
+        /* "NONE" = Do not scale, use Exch-Disp2,u */
+        /* "FIXED" = Use a fixed factor to scale Exch-Disp2,u */
+        /* "NONE" = Use the ratio of Disp20,r and Disp2,u to scale Exch-Disp2,u */
```

Comment:
```suggestion
        /*- Scheme for approximating exchange-dispersion for SAPT-DFT.
        ``NONE`` Use unscaled ``Exch-Disp2,u`` .
        ``FIXED`` Use a fixed factor |sapt__sapt_dft_exch_disp_fixed_scale| to scale ``Exch-Disp2,u`` .
        ``DISP`` Use the ratio of ``Disp2,r`` and ``Disp2,u`` to scale ``Exch-Disp2,u`` . -*/
```

Reply:
> I think you need to clarify in the comment/doc for SAPT_DFT_EXCH_DISP_SCALE_SCHEME that if the user picks "FIXED", then the fixed factor is the one specified by SAPT_DFT_EXCH_DISP_FIXED_SCALE. Then, in that option, you should add to the comment/doc a short literature citation so people know where the scale factor comes from (e.g., a format like e.g., Hesselman et al., JCP 152, 181141 (2016) or whatever).

@CDSherrill the `|sapt__sapt_dft_exch_disp_fixed_scale|` will format into a link to the FIXED_SCALE option. I definitely agree on the JCP link and will leave that for @yxie326 (as well as committing my suggestion).